Cleanup and addition of troubleshooting guide

artillect opened the pull request:
General fixes generally described in commit titles.

In the future, we should use inline css as little as possible and style things within styles.css. Keep things modular as well.

artillect committed:
Cleanup and begin writing troubleshooting article

artillect committed:
Merge from master and troubleshooting guide

artillect committed:
Merge remote-tracking branch 'upstream/master'

artillect committed:
Removed .makegood class in favor of styling things so they're already "good".

Various cleanups

artillect committed:
Fixed favicons and page titles

pogmommy commented:
lmao thanks for fixing the makegood class, that thing was absolute trash and I was meaning to fix it. That was back when i had issues with the navbar conflicting with regular lists.

artillect commented:
Yeah, I kept seeing you using it and figured I might as well fix it. I've renamed note to highlight-note and added a highlight-warn for the red highlighting. I'm gonna work on cleaning up all of the css so we use the minimal subset of what we've already got.
